common/lib/xmodule/xmodule/error_module.py

```diff
@@ -207,8 +208,19 @@ def export_to_xml(self, resource_fs):
             return etree.tostring(root, encoding='unicode')
 
 
-class NonStaffErrorDescriptor(ErrorDescriptor):
+class NonStaffErrorBlock(
+    ErrorBlock,
+):
```

Comment:
Nit: This style of indentation seems unnecessary where there's just a single superclass?